evennia/commands/default/general.py

```diff
@@ -387,48 +387,72 @@ class CmdGet(COMMAND_DEFAULT_CLASS):
     Usage:
       get <obj>
 
-    Picks up an object from your location and puts it in
-    your inventory.
+    Picks up an object from your location and puts it in your inventory.
     """
 
     key = "get"
     aliases = "grab"
-    locks = "cmd:all();view:perm(Developer);read:perm(Developer)"
```

Comment:
I honestly don't remember why this was :thinking: It seems pretty deliberate though :shrug: